src/xrpld/app/tx/detail/AMMWithdraw.cpp

```diff
@@ -406,22 +428,12 @@ AMMWithdraw::applyGuts(Sandbox& sb)
     if (result != tesSUCCESS)
         return {result, false};
 
-    bool updateBalance = true;
```

Comment:
is it more like a helper function? Since `deleteAMMAccount` is in `AMMUtils`, `deleteAMMAccountIfEmpty` is doing one more step by checking if the lpTokenBalance is 0.

Reply:
since `deleteAMMAccount` is in AMMUtils, I think `deleteAMMAccountIfEmpty` is just doing one more step by checking if the lptokenBalance is 0. So it looks like a helper function. Can we keep it in utils?

Reply:
It's updating the view, which with a few exceptions is done in the transactors or View.